notebooks/official/feature_store/sdk-feature-store.ipynb

```diff
@@ -489,19 +490,19 @@
         "id": "h_HmF24mBHv9"
       },
       "source": [
-        "## Terminology and Concept\n",
+        "## Terminology and concept\n",
         "\n",
-        "### Featurestore Data model\n",
+        "### Featurestore data model\n",
         "\n",
         "Vertex AI Feature Store organizes data with the following 3 important hierarchical concepts:\n",
         "```\n",
         "Featurestore -> Entity type -> Feature\n",
         "```\n",
         "* **Featurestore**: the place to store your features\n",
```

Comment:
Lines 501 onwards - capitalize the first word after the colon. E.g. "* **Entity Type**: Under a featurestore..."